scripts/release-tools/kube-burner-configs/berserker-load/connection-load-deployment.yml

```diff
@@ -37,8 +37,6 @@ spec:
             value: "{{ $ip }}"
           - name: BERSERKER__WORKLOAD__ADDRESS
             value: "{{ $berserker_address }}"
-          - name: RUST_LOG
-            value: "debug"
```

Comment:
Is the default logging level 'info' to where this may still be verbose? if so, does it make sense to force a higher level - ie: "error"? (assuming supported)

Reply:
I now use the error log level for all berserker deployments.